packages/design-system/src/components/Autocomplete/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -26,6 +26,7 @@ exports[`Autocomplete renders Autocomplete component 1`] = `
       autocomplete="off"
       class="ds-c-field"
       id="autocomplete_1"
+      labelid="autocomplete_label_2"
```

Comment:
So I removed labelid from the destructuring in `TextField` and it turns out `AutoComplete` has been using this field previously but it was never showing up, I don't understand why it needs to show up but [in our current codebase it is there](https://github.com/CMSgov/design-system/blob/2589f04689d1eec82213ae3c460b502d1b0962c4/packages/design-system/src/components/Autocomplete/Autocomplete.tsx#L264).

Reply:
Could it possibly have been null/undefined before?

Reply:
I suppose? Depends on the use case? It is listed currently in our Autocomplete documentation https://design.cms.gov/components/autocomplete/?theme=core